README.md

```diff
@@ -8,16 +8,61 @@
 
 ## Installation
 
-Install it yourself as:
-
     $ gem install rsgem
 
 We highly suggest to not include `rsgem` in your Gemfile.
 `rsgem` is not a library, and should not affect the dependency tree of your application.
 
 ## Usage
 
-_TODO_
+```
+rsgem new NAME [CI_PROVIDER]
+```
+
+RSGem will solve the following tasks for you:
+
+1. Create a folder for your gem leveraging bundler's defaults. (You need bundler in your system)
+1. Add the following dependencies:
```

Comment:
why all `1.`s instead of 1. 2. 3. etc?

Reply:
The markdown parser converts them to the correct number